app/styles/app.css

```diff
@@ -15,3 +15,23 @@
 .align-middle {
   vertical-align: middle;
 }
+
+.brand-logos > li {
+  border: 1px solid rgba($black, .3);
+  background-image: url("/images/brand/brand-bg.png");
+  border-radius: 4px;
+  padding: 10px 30px;
```

Comment:
this needs to be relational units not pixels

Reply:
Fixed 👍